inc/css/blocks/class-flip-css.php

```diff
@@ -0,0 +1,273 @@
+						'condition' => function( $attrs ) {
+							return isset( $attrs['backBackgroundType'] ) && 'gradient' === $attrs['backBackgroundType'];
+						},
+					),
+					array(
+						'property'       => '--backBackground',
+						'pattern'        => 'url( imageURL ) repeat attachment position/size',
+						'pattern_values' => array(
+							'imageURL'   => array(
+								'value'  => 'backBackgroundImage',
+								'format' => function( $value, $attrs ) {
+									return $value['url'];
+								},
+							),
+							'repeat'     => array(
+								'value'   => 'backBackgroundRepeat',
+								'default' => 'repeat',
+							),
+							'attachment' => array(
+								'value'   => 'backBackgroundAttachment',
+								'default' => 'scroll',
+							),
+							'position'   => array(
+								'value'   => 'backBackgroundPosition',
+								'default' => array(
+									'x' => 0.5,
+									'y' => 0.5,
+								),
+								'format'  => function( $value, $attrs ) {
+									if ( isset( $value['x'] ) && isset( $value['y'] ) ) {
+										return ( $value['x'] * 100 ) . '% ' . ( $value['y'] * 100 ) . '%';
+									}
+									return '50% 50%';
+								},
+							),
+							'size'       => array(
+								'value'   => 'backgroundSize',
+								'default' => 'auto',
+							),
+						),
+						'condition'      => function( $attrs ) {
+							return isset( $attrs['backBackgroundType'] ) && 'image' === $attrs['backBackgroundType'] && isset( $attrs['backBackgroundImage'] ) && isset( $attrs['backBackgroundImage']['url'] );
+						},
+					),
+					array(
+						'property' => '--backVerticalAlign',
+						'value'    => 'backVerticalAlign',
+					),
+					array(
+						'property' => '--padding',
+						'value'    => 'padding',
+						'unit'     => 'px',
+					),
+					array(
+						'property'       => '--boxShadow',
+						'pattern'        => 'horizontal vertical blur color',
+						'pattern_values' => array(
+							'horizontal' => array(
+								'value'   => 'boxShadowHorizontal',
+								'unit'    => 'px',
+								'default' => 0,
+							),
+							'vertical'   => array(
+								'value'   => 'boxShadowVertical',
+								'unit'    => 'px',
+								'default' => 0,
+							),
+							'blur'       => array(
+								'value'   => 'boxShadowBlur',
+								'unit'    => 'px',
+								'default' => 5,
+							),
+							'color'      => array(
+								'value'   => 'boxShadowColor',
+								'default' => '#000',
+								'format'  => function( $value, $attrs ) {
+									$opacity = ( isset( $attrs['boxShadowColorOpacity'] ) ? $attrs['boxShadowColorOpacity'] : 50 );
+									return ( strpos( $value, '#' ) !== false && $opacity < 100 ) ? $this->hex2rgba( $value, $opacity / 100 ) : $value;
+								},
+							),
+						),
+						'condition'      => function( $attrs ) {
+							return isset( $attrs['boxShadow'] ) && ( ! isset( $attrs['isInverted'] ) || ( isset( $attrs['isInverted'] ) && false === $attrs['isInverted'] ) );
```

Comment:
@HardeepAsrani, why did you disable the shadow in frontend when the sides are inverted?

Reply:
@Soare-Robert-Daniel I didn't. You had: https://github.com/Codeinwp/otter-blocks/commit/05fc51293e51da6bad675181beca40e7df596605 - must be for some reason that you can't remember now. You can test it a bit to see if there are any issues this way.